stateless-svc/templates/_volumes.tpl

```diff
@@ -88,10 +88,9 @@ Render container volumeMounts for given context (container or sidecar). Usage:
 
   {{- if eq (include "stateless-svc.hasSidecarMounts" $root) "true" -}}
     {{ "" }}
-  sidecars:
```

Comment:
this is a property that doesn't exist, and was bad copypasta from a comment I was playing with.